packages/docs/developer-resources/dappkit/usage.md

```diff
@@ -69,6 +70,27 @@ let cUSDBalance = cUSDBalanceBig.toString();
 this.setState({ cUSDBalance, isLoadingBalance: false });
 ```
 
+## Checking attestations for the phone number
+
+If the user is using a Valora version that passes the `pepper` that Valora has for a `phone_number`, you can use both pieces of information to determine attestations for the identifier (learn more about the [lightweight identity protocol here](../../celo-codebase/protocol/identity)):
+
+```javascript
+import { PhoneNumberUtils } from '@celo/utils'
+const attestations = await kit.contracts.getAttestations()
+
+const identifier = PhoneNumberUtils.getPhoneHash(dappkitResponse.phoneNumber, dappkitResponse.pepper)
```

Comment:
Looks great! One nit: we are trying to standardize the style in the docs; could you add semi-colons to the end of the lines in this code snippet? Other than that looks good!